### DISCUSSION TITLE
how can i config full syncrhonzation from mysql to clickhouse?

### DISCUSSION THREAD

**Discussion by lddlww** [apache/seatunnel#7811] [Q&A]
i want to sync mysql data to clickhouse with two steps :
step 1: full synchronzation 
step 2: increment synchronzation

but i foud clickhouse sink must be configed with table in seatunnel document, if my mysql database has a lot of tables, how can i do to simplified it?

**Comment by liunaijie** [comment]
you can use variable to define the clickhouse table name, it will be replace with the mysql source table name in runtime
